api/src/api/application_alpha/application_schemas.py

```diff
@@ -0,0 +1,10 @@
+from src.api.schemas.extension import Schema, fields
+
+
+class ApplicationStartRequestSchema(Schema):
+    competition_id = fields.UUID(required=True)
+
+
+class ApplicationStartResponseSchema(Schema):
+    message = fields.String()
```

Comment:
Make sure response schemas are derived from AbstractResponseSchema - don't need to specify anything but data with that.
```suggestion
class ApplicationStartResponseSchema(AbstractResponseSchema):

```